src/Microsoft.OpenApi/Writers/OpenApiWriterBase.cs

```diff
@@ -470,16 +470,16 @@ public void WriteJsonSchemaWithoutReference(IOpenApiWriter writer, JsonSchema sc
             writer.WriteProperty(OpenApiConstants.Title, schema.GetTitle());
 
             // multipleOf
-            writer.WriteProperty(OpenApiConstants.MultipleOf, schema.GetMultipleOf());
+            writer.WriteProperty(OpenApiConstants.MultipleOf, schema.GetOpenApiMultipleOf());
```

Comment:
Are we forced to prefix with OpenAPI to differentiate from the jsonschema library methods?
Is there any reason why we can't use that library's methods?

Reply:
Yes prefixing with OpenApi helps to distinguish our custom implementation from the JsonSchema.NET library's.
Also we're overriding the library's methods because the `MultipleOf` keyword is defined as a decimal type and we need to replace that and use double for us to support parsing schema values exceeding decimal's ranges.

Reply:
Wouldn't it be better to PR upstream instead of maintaining our own code in this case?

Reply:
We've decided to hold off on this PR and any work around it till v2.1